src/renderers.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { DEFAULT_RENDERERS } from 'myst-to-react';
+import { MermaidNodeRenderer } from '@myst-theme/diagrams';
+import { NodeRenderer } from '@myst-theme/providers';
+import { InlineRenderer } from './inlineExpression';
+
+export const renderers: Record<string, NodeRenderer> = {
```

Comment:
there are a lot of _renderers_ flying around! these are mystRenderers specifically here though it seems

Reply:
Yeah, I think there is a wider clean up for the whole repo as we start to get a bit more complicated roles and parsing structure in this repo.

I am inclined to kick this down the road though!